### PR TITLE
PHP: attempt to add v1.21.4 to interop matrix

### PR DESCRIPTION
**Pull request by stanley-cheung**

### CODE REVIEW

**Comment by jtattermusch** [comment]
https://github.com/grpc/grpc/pull/19230 would need to be backported to make this PR work.

**Comment by stanley-cheung** [comment]
@jtattermusch This is ready for another review. The Adhoc interop matrix job was succesful - I manually triggered another build, PTAL. Thanks.

File: tools/interop_matrix/client_matrix.py [hunk 1]
```diff
@@ -281,8 +281,7 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
         ('v1.16.0', ReleaseInfo(testcases_file='php__v1.0.1')),
         ('v1.17.1', ReleaseInfo(testcases_file='php__v1.0.1')),
         ('v1.18.0', ReleaseInfo()),
-        # TODO:https://github.com/grpc/grpc/issues/18264
```
Comment (jtattermusch) [inline tools/interop_matrix/client_matrix.py]: Please leave a comment why v1.19 an v1.20 are missing so it's obvious this is not a mistake.

Comment (stanley-cheung) [reply tools/interop_matrix/client_matrix.py]: Done.

**Pull request merged**